crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1454,6 +1457,7 @@ pub enum SuiEndOfEpochTransactionKind {
     AuthenticatorStateExpire(SuiAuthenticatorStateExpire),
     RandomnessStateCreate,
     CoinDenyListStateCreate,
+    BridgeStateCreate,
```

Comment:
I wonder if there's a better way for creating more fixed objects than adding a new txn type each time...maybe something like a "post-upgrade" move call that can be a regular move call without needing the extra boilerplate?

Reply:
Do we call init during system txns? I totally forget. 
We don't for system package upgrades (or any upgrades) which is otherwise an issue. 

But yeah, I could go either way. Like, I could imagine having something like `System(ProgrammableTransaction)` that just lets you do what you want (which is basically already happening with the create functions, just static vs dynamic)

No strong opinions here.

Reply:
We do call `init` when publishing a brand new system package, but not otherwise (i.e. not during publish).